USAGE.md

```diff
@@ -0,0 +1,134 @@
+environment for ad-hoc use.  The below assumes you have setup your `ansibullbot.cfg` as described in the previous
+section, and made the necessary edits to `ansibullbot/triagers/ansible.py`.  Any repositories passed with `--repo`
+must be enabled in your ansibullbot.cfg file.
+
+```shell script
+$ cd collection_bot/
+$ python2.7 -m virtualenv .ansibot
+$ source .ansibot/bin/activate
+$ python2.7 -m pip install -r requirements.txt
+$ python2.7 -m pip install epdb  ## Ansibullbot has breakpoints, epbd must be installed if one it hit
+$ python2.7 triage_ansible.py --repo ansible-collections/mycollection --repo ansible-collections/some_other_repo
+```
+
+See `python2.7 triage_ansible.py --help` for more options, including daemonize, dry-run, and debug flags.
+
+Deploying to AWS
+----------------
+
+This repo ships with playbooks that, along with the references submodules, will deploy the bot to an AWS EC2 instance
+where it can be run as a daemon along with a public status webpage and reports. This step is optional, the bot can be run
+from any host the operator prefers.
+
+The roles and playbooks used for deploying and managing the bot instance contain a number of variables and values
+that will need to be customized for your deployment.  You will want to create an `inventory.yml` file with the
+following vars, customized as needed (this method assumes example.com uses Route53 for DNS).
+
+Be sure that the ssh key for the user you are currently running as is the first entry in the botinstance_ssh_keys
+dictionary.  This is the key that will be provided to ec2_instance and used for bot installation tasks.
+
+```yaml
+all:
+  vars:
+    botinstance_name: ansibullbot-mycollection
+    botinstance_region: us-west-1
+    botinstance_volume_size: 200
+    botinstance_type: t3.xlarge
+    botinstance_ssh_keys:
+      - name: mykey
+        key_material: ssh-rsa 12345567890aaaaaaaaaaaa...
+    ansibullbot_fqdn: ansibullbot-mycollection.example.com
+    botinstance_dns_entries:
+      - type: CNAME
+        record: ansibot-mycollection.example.com
+        value: ansibullbot-mycollection.example.com
+    
+      - type: CNAME
+        record: ansiblebot-mycollection.example.com
+        value: ansibullbot-mycollection.example.com
+    
+      - type: A
+        command: create
+        record: ansibullbot-cloud.enxample.com
+        value: "{{ _elastic_ip.public_ip }}"
+
+    ansibullbot_repos:  ## List
+      - ansible-collections/mycollection
+      - ansible-collections/some_other_repo
+
+    ansibullbot_options:
+      - daemonize
+      - daemonize_interval={{ ansibullbot_daemonize_interval }}
+      - debug
+      - force
+      - logfile {{ ansibullbot_log_path }}
+      - resume
+      - skip_no_update
+      - skip_no_update_timeout
+      - verbose
+      {% for collection in ansibullbot_repos %}
+      - repo "{{ repo }}"
+      {% endfor %}
```

Comment:
I think it would be better if this was in a config file rather than 1) using an in-line for loop here and 2) making a pretty long command in the unit file.

Reply:
Yeah right now `C.DEFAULT_REPOS` (and the static `REPOS` that it replaces) and `self.repo` have a lot of overlap, it wasn't entirely clear to me what the intent of having these be separate values was so I erred on the side of preserving that logic.

The cloud bot currently runs against 3 (soon to be 4) collections.  This does let me log on to the bot and adhoc run against just one collection for troubleshooting or whatever without having to modify the config file.  I'm not sure how much value that actually adds though in practice.